mod-orders/schemas/composite_purchase_order.json

```diff
@@ -58,9 +58,10 @@
         "Ongoing"
       ]
     },
-    "owner": {
-      "description": "The team owner of this purchase order",
-      "type": "string"
+    "acquisitionsUnit": {
+      "description": "The acquisitions unit of this purchase order",
```

Comment:
Better to use description from [Acquisitions Interface Fields](https://docs.google.com/spreadsheets/d/1F88PoSnEKD7oFnza1ZuOgzAgKnK95XOyTQzxyww3tZA/edit#gid=1408113181&range=P13)
> Which acquisitions unit (Team) owns this order (and is responsible for dealing with its distribution).

Reply:
Done